song_rater/tests.py

```diff
@@ -29,3 +29,16 @@ def test_add_song_post(self):
     def test_add_song_incomplete_post(self):
         response = self.client.post('/add-song', {'artist': 'Orange'})
         self.assertEqual(response.status_code, 200)
+
+
+class SongListTestCase(TestCase):
+    def setUp(self):
+        song = Song.objects.create(artist='Apple', title='Pie')
+        song.ratings.create(rating=3)
+
+    def test_song_list(self):
+        response = self.client.get('/song-list')
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'Apple', count=1)
+        self.assertContains(response, 'Pie', count=1)
+        self.assertContains(response, '3', count=1)
```

Comment:
What are  you testing when you look in the page for the number `3` ? 

I presume you are testing for an average rating. Maybe state so in a small comment.

Reply:
Thank you for finding this, I added a comment for clarification. No need for magic numbers